spdm.rflx

```diff
@@ -299,7 +304,55 @@ package SPDM is
                   and (Mut_Auth_Cap = False or Encap_Cap = True);
       end message;
 
-   -- 10.11 Measurement block
+   -- 10.11 GET_MEASUREMENTS request and MEASUREMENTS response message
+
+   --  The value 0 queries the responder for the total number of measurements
+   --  available. The value 255 requests all measurements. The values 1 to 254
+   --  request the measurement at the index corresponding to that value.
+   type Measurement_Operation is
+      (Get_Number_Of_Measurements => 0,
+       Get_All_Measurements => 255)
```

Comment:
Please align the arrows.

Reply:
Done in https://github.com/Componolit/RecordFlux-specifications/pull/82/commits/bf8586a380aca0390d1e30aa26347ab9e220ed06.